tools/add_font.py

```diff
@@ -21,6 +50,9 @@
 flags.DEFINE_integer('min_pct_ext', 10,
                      'What percentage of subset codepoints have to be supported'
                      ' for a -ext subset.')
+flags.DEFINE_boolean('update', False,
```

Comment:
I'm going to merge because it looks fine, feel free to send a PR if you decide modifying the flag is worthwhile.